cyberbrain-vsc/src/rpc_server.ts

```diff
@@ -37,6 +39,11 @@ export class RpcServer {
               );
             }
           }
+          if (message.command === "Interaction behaivor") {
+            if (message.context) {
```

Comment:
We either log an error when there's no `context`, or remove this if check because we know `context` always exists.

Reply:
`context` does not exist when the message command is `webview ready`. although we knew that when the command is `interaction behaivor`, the `context` always exist, the computer does not know about that and will have lint for error. Maybe just keep the judgement statement, and also I add a log for catching errors.